### PR TITLE
Bot API 7.6

### PR DESCRIPTION
**Pull request by coder2020official**
- [x] Added the classes [PaidMedia](https://core.telegram.org/bots/api#paidmedia), [PaidMediaInfo](https://core.telegram.org/bots/api#paidmediainfo), [PaidMediaPreview](https://core.telegram.org/bots/api#paidmediapreview), [PaidMediaPhoto](https://core.telegram.org/bots/api#paidmediaphoto) and [PaidMediaVideo](https://core.telegram.org/bots/api#paidmediavideo), containing information about paid media.
- [x] Added the method [sendPaidMedia](https://core.telegram.org/bots/api#sendpaidmedia) and the classes [InputPaidMedia](https://core.telegram.org/bots/api#inputpaidmedia), [InputPaidMediaPhoto](https://core.telegram.org/bots/api#inputpaidmediaphoto) and [InputPaidMediaVideo](https://core.telegram.org/bots/api#inputpaidmediavideo), to support sending paid media.
- [x] Documented that the methods [copyMessage](https://core.telegram.org/bots/api#copymessage) and [copyMessages](https://core.telegram.org/bots/api#copymessages) cannot be used to copy paid media.
- [x] Added the field can_send_paid_media to the class [ChatFullInfo](https://core.telegram.org/bots/api#chatfullinfo).
- [x] Added the field paid_media to the classes [Message](https://core.telegram.org/bots/api#message) and [ExternalReplyInfo](https://core.telegram.org/bots/api#externalreplyinfo).
- [x] Added the class [TransactionPartnerTelegramAds](https://core.telegram.org/bots/api#transactionpartnertelegramads), containing information about Telegram Star transactions involving the Telegram Ads Platform.
- [x] Added the field invoice_payload to the class [TransactionPartnerUser](https://core.telegram.org/bots/api#transactionpartneruser), containing the bot-specified invoice payload.
- [x] Changed the default opening mode for [Direct Link Mini Apps](https://core.telegram.org/bots/webapps#direct-link-mini-apps).
- [x] Added support for launching Web Apps via t.me link in the class [MenuButtonWebApp](https://core.telegram.org/bots/api#menubuttonwebapp).
- [x] Added the field section_separator_color to the class [ThemeParams](https://core.telegram.org/bots/webapps#themeparams).

### CODE REVIEW

**Comment by coder2020official** [comment]
@badiboy done with the update i think, need your review

**Comment by Badiboy** [comment]
Will take a look

File: telebot/types.py [hunk 1]
```diff
@@ -10479,4 +10479,111 @@ def de_json(cls, json_string):
...
+        self.thumbnail = thumbnail
+        self.width = width
+        self.height = height
+        self.duration = duration
+        self.supports_streaming = supports_streaming
+
+    def to_dict(self):
+        data = super().to_dict()
+        if self.thumbnail:
+            data['thumbnail'] = self.thumbnail.to_dict()
```
Comment (Badiboy) [inline telebot/types.py:10651]: Thumbnail is "InputFile or String"

Should be

if self.thumbnail:
            data['thumbnail'] = self.thumbnail

**Comment by Badiboy** [comment]
To the review (in progress): 

convert_input_media_array should also isinstance the InputPaidMedia

File: telebot/types.py [hunk 2]
```diff
@@ -10479,4 +10479,111 @@ def de_json(cls, json_string):
...
+    def __init__(self, type, media, **kwargs):
+        self.type = type
+        self.media = media
+
+    def to_json(self):
+        return json.dumps(self.to_dict())
+    
+    def to_dict(self):
+        data = {}
+        data['type'] = str(self.type)
```
Comment (Badiboy) [inline telebot/types.py:10577]: Why (str) is here?

File: telebot/types.py [hunk 3]
```diff
@@ -10479,4 +10479,111 @@ def de_json(cls, json_string):
...
+        data['media'] = str(self.media)
```
Comment (Badiboy) [inline telebot/types.py:10578]: Why (str) is here?

File: telebot/types.py [hunk 4]
```diff
@@ -10479,4 +10479,111 @@ def de_json(cls, json_string):
...
+        data = super().to_dict()
+        if self.thumbnail:
+            data['thumbnail'] = self.thumbnail.to_dict()
+        if self.width:
+            data['width'] = self.width
+        if self.height:
+            data['height'] = self.height
+        if self.duration:
+            data['duration'] = self.duration
+        if self.supports_streaming:
```
Comment (Badiboy) [inline telebot/types.py:10658]: if self.supports_streaming is not None:

**Comment by Badiboy** [comment]
> need your review

Done.

**Comment by coder2020official** [comment]
@badiboy fixed your addressed points. Also, manually tested the send_paid_media method - it worked after changing InputPaidMedia to behave like InputMedia

**Pull request merged**